README.md

```diff
@@ -1,9 +1,12 @@
-# dwyl hq
-:red_circle: open kimono
+# dwyl
 
-# Why?
+DWYL LTD is incorporated and registered in England and Wales with company number 09525434 and registered office is at DWYL LTD, 104 Mary Datchelor Close, London, SE5 7DY, United Kingdom.
```

Comment:
I agree it's important info, particularly the registered company no. and address, I'm just requesting we move the info around to reflect what we want our audience to see first - most people who get here don't know dwyl as anything more than an awesome online resource!